lib/LLVMIRCodeGen/BundleSaver.cpp

```diff
@@ -43,6 +43,60 @@ using llvm::cast;
 using llvm::dyn_cast;
 using llvm::isa;
 
+enum BundleApiType {
+  /// Dynamic bundle API (default) with the following features:
+  /// - the weights are exported in a binary file which are assumed
+  ///   to be loaded dynamically at run-time.
+  /// - the memory layout information (bundle configuration) is only
+  ///   available at run-time and therefore allows ONLY dynamic memory
+  ///   allocaton.
+  Dynamic,
+  /// Static bundle API with the following features:
+  /// - the weights are exported in a binary file but and also in a
+  ///   text file (C array format) suitable to include at compile-time.
+  /// - the memory layout information (bundle configuration) is available
+  ///   at compile-time through macros printed in the header file and thus
+  ///   allows also static memory allocation.
+  /// - this API is suitable for low end devices with no file system or OS
+  ///   (bare-metal).
+  Static,
+};
+
+namespace {
+llvm::cl::OptionCategory loaderCat("Bundle Options");
```

Comment:
Why is it called `loaderCat`? May be `bundleCat` or `bundleSaverCat` is a better choice?

Reply:
I have changed it bundleSaverCat.